meta-mel/conf/distro/include/jobs-heuristic.inc

```diff
@@ -0,0 +1,95 @@
+# Set bitbake threads and make jobs based upon available memory per CPU core.
+# Based on https://elinux.org/images/d/d4/Goulart.pdf, a talk done by Mario Domenech
+# Goulart and Mikko Rapeli from BMW CAR IT GMBH.
+#
+# MEM_PER_CPU_MINIMUM,THREADS,JOBS,ACTION
+JOBS_HEURISTIC_THRESHOLDS ?= "\
+    7.9,cpus,cpus*1.5, \
```

Comment:
for 8G or greater per cpu the study uses threads=cpus pmake=cpus, we've pmake=cpus*1.5 here.